sdk/digitaltwins/arm-digitaltwins/package.json

```diff
@@ -3,15 +3,15 @@
   "sdk-type": "mgmt",
```

Comment:
Could we add one testing for this rp? Maybe the simplest one is listalloperations?